GitHub_LearningTests/Generic/OnboardingPersistancy/Resources/OnboardingResourceTests.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  OnboardingResourceTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 18.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import XCTest
+import CleanCore
+import CleanPlatform
+@testable import GitHub_Learning
+
+class OnboardingResourceTests: XCTestCase {
+    private var sut: OnboardingResourceImpl!
+    
+    private func setUpTests(
+        localStorage: LocalStorage = LocalStorageDummy()
+    ) {
+        sut = OnboardingResourceImpl(localStorage: localStorage)
+    }
+    
+    func test_givenOnboardingResource_whenloadIsOnboardingFinished_thenDataAreLoaded() throws {
+        // Given
+        let localStorage = LocalStorageDummy()
+        setUpTests(localStorage: localStorage)
+        
+        // When
+        let _ = try sut.loadIsOnboardingFinished()
```

Comment:
V tomto testu bych radsi predaval dummy co ma vracet a pak to kontroloval s tim, co ti vratit sut.